data/meson.build

```diff
@@ -50,15 +50,23 @@ if get_option('systemd')
     unit_conf.set('bindir', bindir)
 
     configure_file(
-        input: 'gala-x11.service.in',
-        output: 'gala-x11.service',
+        input: 'io.elementary.gala@x11.service.in',
+        output: 'io.elementary.gala@x11.service',
```

Comment:
Can we rename during build here to avoid extra long filenames in the source tree? Makes it easier to switch files in Code's sidebar for example if they don't all begin with the same prefix

Reply:
stripped the io.elementary. prefix